src/mqtt.cpp

```diff
@@ -33,6 +33,7 @@ static bool connecting = false;
 uint8_t claimsVersion = 0;
 uint8_t overrideVersion = 0;
 uint8_t scheduleVersion = 0;
+uint32_t config_version_mqtt = 1;
```

Comment:
can you use the same variable format  as for claimsVersion, overrideVersion, ... above ? 
=> configVersion = 0;